### ISSUE TITLE
Paper Trader Won't Load, Market Watcher Fine

### ISSUE THREAD

**Issue by tkreis2** [askmike/gekko#1260]
* **I'm submitting a ...**
[X ] bug report
[ ] feature request
[ ] question about the decisions made in the repository

* **Action taken** (what you did)
I tried to start a paper trader on the Poloniex exchange in the UI using 1) MACD strategy and then 2) a custom strategy I've developed. I was able to use both last week.

* **Expected result** (what you hoped would happen)
The paper trader would take a minute to load and then begin performing trades according to the selected strategy.

* **Actual result** (unexpected outcome)
The paper trader does not load. The loading sign under "Runtime" in the UI continues for 40+ minutes. It appears that getCandles isn't being called.

* **Other information** (e.g. detailed explanation, stacktraces, related issues, suggestions how to fix, links for us to have context, eg. stackoverflow, etc)
-I'm using Windows 10 os and Gekko v0.5.9.
-When I tested this on a friend's Windows 7 machine, it worked.
-I currently see this error in the browser developer tools, which I've seen mentioned elsewhere: 
Uncaught TypeError: Cannot read property 'toLowerCase' of undefined
    at Oe.e.$emit (build.js:6)
    at a.s.onmessage (build.js:6)
    at HTMLDivElement.<anonymous> (reconnecting-websocket.min.js:2)
    at WebSocket.h.onmessage (reconnecting-websocket.min.js:2)
However, these errors occurred when I used the program last week, but I was still able to run my paper trader successfully.
-The market watcher is working and pulling in data.
-In the log files, the paper trader only reaches this point:
For MACD:
2017-10-28 21:46:51 (INFO):	Setting up Gekko in realtime mode
2017-10-28 21:46:51 (INFO):	
2017-10-28 21:46:51 (INFO):	Setting up:
2017-10-28 21:46:51 (INFO):		 Candle writer
2017-10-28 21:46:51 (INFO):		 Store candles in a database
2017-10-28 21:46:51 (INFO):	

2017-10-28 21:46:51 (INFO):	Setting up:
2017-10-28 21:46:51 (INFO):		 Trading Advisor
2017-10-28 21:46:51 (INFO):		 Calculate trading advice
2017-10-28 21:46:51 (INFO):		 Using the strategy: MACD
2017-10-28 21:46:51 (INFO):	

2017-10-28 21:46:51 (INFO):	Setting up:
2017-10-28 21:46:51 (INFO):		 Paper Trader
2017-10-28 21:46:51 (INFO):		 Paper trader that simulates fake trades.
2017-10-28 21:46:51 (INFO):	

2017-10-28 21:46:51 (INFO):	Setting up:
2017-10-28 21:46:51 (INFO):		 Performance Analyzer
2017-10-28 21:46:51 (INFO):		 Analyzes performances of trades
2017-10-28 21:46:51 (INFO):	

For my custom strategy:
2017-10-28 21:32:07 (INFO):	Setting up Gekko in realtime mode
2017-10-28 21:32:07 (INFO):	
2017-10-28 21:32:07 (INFO):	Setting up:
2017-10-28 21:32:07 (INFO):		 Candle writer
2017-10-28 21:32:07 (INFO):		 Store candles in a database
2017-10-28 21:32:07 (INFO):	

2017-10-28 21:32:07 (INFO):	Setting up:
2017-10-28 21:32:07 (INFO):		 Trading Advisor
2017-10-28 21:32:07 (INFO):		 Calculate trading advice
2017-10-28 21:32:07 (INFO):		 Using the strategy: custom
2017-10-28 21:32:07 (DEBUG):	Reached init // I added this print statement as a check.
2017-10-28 21:32:07 (INFO):	

2017-10-28 21:32:07 (INFO):	Setting up:
2017-10-28 21:32:07 (INFO):		 Paper Trader
2017-10-28 21:32:07 (INFO):		 Paper trader that simulates fake trades.
2017-10-28 21:32:07 (INFO):	

2017-10-28 21:32:07 (INFO):	Setting up:
2017-10-28 21:32:07 (INFO):		 Performance Analyzer
2017-10-28 21:32:07 (INFO):		 Analyzes performances of trades
2017-10-28 21:32:07 (INFO):	

It appears that Gekko isn't reaching the update function. I have added my parameters in the custom.toml file for my own strategy.

- I have tried uninstalling and reinstalling Gekko, but have had the same result.

- I have also tried to run the paper trader with other exchanges including bitfinex but am running into the same error.

I would really appreciate any help! Thank you.

**Comment by askmike** [comment]
What is the candle size & history size you configured?

**Comment by tkreis2** [comment]
I am doing some testing, so my candle size is 1 minute and my history size is 1. I also tested a 2 minute candle size with a history of 2, and a 10 minute candle size with a history of 1.5 and ran into the same issue. 
I put console.log statements in several files to try to pinpoint where the error is happening. I can see these statements in: budfox.js, candleCreator.js, loadCandles/child.js, candleManager.js, gekkoStream.js, paperTrader.js, tradingAdvisor.js, pipeline.js. 
I do not see the debug statements in the candleLoader.js and candleBatcher.js files, and I only saw the statement in getCandles.js 1 time.
I've included screenshots to show you an example of what I see in the console (sorry I couldn't get all the content into 1 shot):
![image](https://user-images.githubusercontent.com/31675216/32155718-82159edc-bd0f-11e7-86cf-45c951be5ae3.png)
![image](https://user-images.githubusercontent.com/31675216/32155731-9375b7ac-bd0f-11e7-8323-4ac806840dae.png)

Thank you for responding!
